### DISCUSSION TITLE
Dedicated Metrics tab

### DISCUSSION THREAD

**Discussion by steff-mat** [coollabsio/coolify#4233] [Feature Requests]
As of latest version, Metrics is back, and is perfect to be able to monitor what's happening without having to manually deal with the configuration.

While the current implementation is ok, would be a huge improvement to have a dedicated metrics tab, where all the containers can be monitored, without having to manually check one by one.

Also, I have observed, as for example, for a PocketBase deployment, there's no metrics, so I'm unsure if that's a general issue/local bug or metrics is available only for services manually deployed and not created from the list provided within Coolify.

**Comment by pforques** [comment]
Hey, I agree the metrics tabs idea. That is not mandatory, but significant enough to be implemented.

It would also be nice if we could have a network chart along the CPU/RAM chart !

**Comment by towfiqi** [comment]
+1

**Comment by iniTwakkie** [comment]
+1

**Comment by abuharsky** [comment]
+100

**Comment by SultanDino** [comment]
bump++
